sources/lib/frontend/frontend.ml

```diff
@@ -330,9 +330,10 @@ module Make(SAT : Sat_solver_sig.S) : S with type sat_env = SAT.t = struct
         report_loc Loc.dummy time steps
 
   let print_status status steps =
-    if Options.output_smtlib () then print_status_output_smtlib status steps
-    else if Options.output_native () then print_status_valid_mode status steps
-    else assert false           (* will be useful for szs *)
+    let open Options in
```

Comment:
you don't need this module opening I think, the type inference is smart enough to know the return type of `Options.output_format ()` if I recall correctly.

Reply:
Done